scripts/setup-centos8.sh

```diff
@@ -189,14 +207,24 @@ function install_velox_deps {
 
 (
   if [[ $# -ne 0 ]]; then
+    # Activate gcc9; enable errors on unset variables afterwards.
+    source /opt/rh/gcc-toolset-9/enable || exit 1
+    set -u
     for cmd in "$@"; do
       run_and_time "${cmd}"
     done
+    echo "All specified dependencies installed!"
   else
+    if [ "${INSTALL_PREREQUISITES:-Y}" == "Y" ]; then
```

Comment:
You might want to add an else saying skipping installation of build prerequisites because of flag so and so..

Reply:
Done